dpr_reader.py

```diff
@@ -105,26 +107,33 @@ def rank(
         if not docs:
             return None
 
-        traversal_paths = parameters.get('traversal_paths', self.traversal_paths)
+        traversal_paths = parameters.get(
+            'traversal_paths',
+            self.traversal_paths
+        )
         batch_size = parameters.get('batch_size', self.batch_size)
 
-        for doc in docs.traverse_flat(traversal_paths):
-            if not getattr(doc, 'text'):
-                continue
-
+        for doc in docs.traverse_flat(traversal_paths, filter_fn=lambda x: bool(x.text)):
             new_matches = []
 
             doc_arr = DocumentArray([doc])
-            # match_batches_generator = doc_arr.batch(
-            #     traversal_paths=['m'],
-            #     batch_size=batch_size,
-            #     require_attr='text',
-            # )
-            match_batches_generator = doc_arr.traverse_flat(traversal_paths=['m'], filter_fn= lambda x: bool(x)).batch(batch_size=batch_size)
+
+            match_batches_generator = doc_arr.traverse_flat(
+                traversal_paths=['m'],
+                filter_fn=lambda x: bool(x.text)
+            ).batch(batch_size=batch_size)
+
             for matches in match_batches_generator:
-                question, titles = self._prepare_inputs(doc_arr[0].text, matches)
+                question, titles = self._prepare_inputs(
+                    doc_arr[0].text,
```

Comment:
Why do we break lines here now?

Reply:
We use `black` for styling, not autpep8

Reply:
I see. I should also migrate git hooks from the executors.